spec.md

```diff
@@ -219,8 +219,9 @@ help intermediate gateways determine how to route the events.
 * Type: `String`
 * Description: ID of the event. The semantics of this string are explicitly
   undefined to ease the implementation of producers. Enables deduplication.
+  See the [Primer](primer.md#id) for more information.
 * Examples:
-  * A database commit ID
```

Comment:
If you wanted to replay transactions from a database, you would want to use the commit ID in the cloudevent, otherwise you do not have a method for detecting duplicated events and the ID in the context the cloudevent is irrelevant.

Reply:
I think commit ID is a valid choice for some cases, however, if that DB commit results in more than one event then it's not a good choice. So, I wanted to avoid getting into those nuances in this section and decided to leave that for the primer. I think UUID is a more clear example.